bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -609,6 +609,10 @@ impl Room {
         })))
     }
 
+    pub fn pin_user_identity(&self, user_id: &str) -> Result<(), ClientError> {
```

Comment:
could do with some documentation?

Reply:
We don't document in the FFI layer normally - is there some reason why this one requires it?

Reply:
Actually we should (and have) because they get properly forwarded to the bindings.

Reply:
OK, update force-pushed with docs included